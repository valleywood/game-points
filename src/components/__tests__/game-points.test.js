import React from 'react';
import GamePoints from '../game-points';
import ReactTestRenderer from 'react-test-renderer';

jest.mock('../../containers/game-item-links', () => 'GameItemLinks');
jest.mock('../player', () => 'Player');

it('renders without crashing', () => {
  const renderer = ReactTestRenderer.create(<GamePoints />);
  expect(renderer.toJSON()).toMatchSnapshot();
});
