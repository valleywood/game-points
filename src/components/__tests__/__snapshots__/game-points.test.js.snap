// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="points"
>
  <div
    className="game-items"
  >
    <header
      className="header"
    >
      <h1>
        Kahoot! POINTS
      </h1>
    </header>
    <GameItemLinks />
  </div>
  <div
    className="player"
  >
    <Player />
  </div>
</div>
`;
